Formula/llvm.rb

```diff
@@ -454,11 +441,15 @@ def caveats
   end
 
   test do
-    llvm_version = Version.new(Utils.safe_popen_read(bin/"llvm-config", "--version").strip)
-    soversion = llvm_version.major.to_s
-    soversion << "git" if head?
+    llvm_version = Utils.safe_popen_read(bin/"llvm-config", "--version").strip
+    soversion = Version.new(llvm_version).major.to_s
+
+    if llvm_version.end_with?("git")
```

Comment:
```suggestion
    llvm_version = Version.new(Utils.safe_popen_read(bin/"llvm-config", "--version").strip)
    soversion = llvm_version.major.to_s

    if llvm_version.to_s.end_with?("git")
```
This will allow us to keep `llvm_version.major` in the OpenMP test below, and will result in a smaller diff.